WalletWasabi/Http/Models/UriScheme.cs

```diff
@@ -3,13 +3,9 @@ namespace WalletWasabi.Http.Models
 	// https://tools.ietf.org/html/rfc7230#section-2.7.3
 	// The scheme and host are case-insensitive and normally provided in lowercase;
 	// all other components are compared in a case-sensitive manner.
-#pragma warning disable IDE1006 // Naming Styles
-
```

Comment:
Is it ok to remove this? Is this unnecessary?

Reply:
I think it is unnecessary, when I removed it I didn't get any suggestion.